src/ngraph/runtime/backend_manager.cpp

```diff
@@ -32,12 +32,18 @@
 using namespace std;
 using namespace ngraph;
 
+#ifdef NGRAPH_STATIC_LIB_ENABLE
+#define DLERROR() ""
+#else
 #ifdef _WIN32
 #define CLOSE_LIBRARY(a) FreeLibrary(a)
 #define DLSYM(a, b) GetProcAddress(a, b)
+#define DLERROR() dlerror()
```

Comment:
dlerror() is linux not windows. Linux has the empty macro and windows has dlerror()